File: factions/neutral/jester.py
from game import Game, Player
from factions import Faction


class JesterNeutral(Faction):
    name = 'Neutral'
    id = 'neutral.jester'
    win_con = 'Get lynched.'

    def has_won_independent(self, player: Player):
        if not player.alive and player.death_reason.startswith('lynched'):
            return True
        return False

File: src/lib/mafia/factions/neutral/Jester.ts
import Faction from '@mafia/structures/Faction';
import type Player from '@mafia/structures/Player';

export default class JesterFaction extends Faction {
	public name = 'Jester';
	public winCondition = 'game/factions:jesterWinCondition';
	public independent = true;

	public hasWonIndependent(player: Player) {
		return !player.isAlive && player.deathReason.includes('eliminated');
	}
}
